lib/ansible/module_utils/aws/waiters.py

```diff
@@ -317,6 +330,12 @@ def rds_model(name):
         core_waiter.NormalizedOperationMethod(
             ec2.describe_vpn_gateways
         )),
+    ('EC2', 'vgw_detached'): lambda ec2: core_waiter.Waiter(
```

Comment:
Super minor nit - this could be named more like the one above, i.e. vpn_gateway_detached. It's a little more similar to how boto3 usually names their waiters.

Reply:
naming things is hard :)